iOS/DuckDuckGo/DataBrokerProtectionDebugViewController.swift

```diff
@@ -88,6 +145,37 @@ final class DataBrokerProtectionDebugViewController: UITableViewController {
             let row = DatabaseRows(rawValue: indexPath.row)
             cell.textLabel?.text = row?.title
 
+            switch row {
+            case .databaseBrowser, .saveProfile, nil: break
+            case .deviceIdentifier:
+                cell.detailTextLabel?.font = UIFont.monospacedSystemFont(ofSize: 17, weight: .regular)
+                cell.detailTextLabel?.text = DataBrokerProtectionSettings.deviceIdentifier
+            }
+
+        case .healthOverview:
+            switch self.healthOverview {
+            case .loading: cell.textLabel?.text = "Loading..."
+            case .runPrerequisitesNotMet(let hasAccount, let hasEntitlement, let hasProfile):
+                if indexPath.row == 0 {
+                    cell.textLabel?.text = "Privacy Pro Account"
+                    cell.detailTextLabel?.text = hasAccount ? "✅" :"❌"
+                } else if indexPath.row == 1 {
+                    cell.textLabel?.text = "PIR Entitlement"
+                    cell.detailTextLabel?.text = hasEntitlement ? "✅" :"❌"
+                } else if indexPath.row == 2 {
+                    cell.textLabel?.text = "Profile Saved In DB"
+                    cell.detailTextLabel?.text = hasProfile ? "✅" :"❌"
+                } else {
+                    fatalError("Expected 3 rows for the health overview")
+                }
+            case .runPrerequesitesMet(let jobScheduled):
+                if jobScheduled {
+                    cell.textLabel?.text = "✅ PIR will run some time after device is locked and connected to power"
+                } else {
+                    cell.textLabel?.text = "❌ Restart the app to schedule PIR"
```

Comment:
This should also check whether the app has permission in the iOS Privacy settings. Otherwise, I'm also open to a more appropriate message here - I don't think this will really show up in many cases though, since the job is scheduled immediately.

Reply:
Added a check for the background refresh permission, I'll merge this when CI is green 🎉